Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -135,10 +158,11 @@ void DoState(PointerWrap& p)
 
 static void ResetRegisters()
 {
-  memset(ppcState.ps, 0, sizeof(ppcState.ps));
-  memset(ppcState.sr, 0, sizeof(ppcState.sr));
-  memset(ppcState.gpr, 0, sizeof(ppcState.gpr));
-  memset(ppcState.spr, 0, sizeof(ppcState.spr));
+  std::fill(std::begin(ppcState.ps), std::end(ppcState.ps), PairedSingle{});
```

Comment:
can't these all just be `x = {};`?

Reply:
They aren't std::array instances (and can't be, since the JITs use `offsetof` to perform static assertions or get structure offsets, so they have to be manually filled like this.